.github/workflows/integrate.yaml

```diff
@@ -35,6 +35,14 @@ jobs:
           path: training-operator-cov_html
         if: failure()
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    with:
+      charm-path: .
+      model: kubeflow
+      channel: latest/edge
```

Comment:
After reviewing the README, I wonder if we should deploy latest/edge when we say this version of the module is compatible with 1.8. My main concern with that is that we are testing a version but documenting another one, which may be inconsistent. Right now there are no differences between 1.8/edge, 1.8/stable and latest/edge, but if I introduce a new relation tomorrow, then the module won't be compatible with 1.8 anymore, and we'll only be testing latest/edge. wdyt about this scenario?

Reply:
The rationale here is that `latest/edge` is >= 1.8 and since we 're saying that this is compatible with >=1.8, then testing is done with a documented version. I preferred to go with `latest/edge` due to merging this in `main`, and it felt weird to test with a track version.

>but if I introduce a new relation tomorrow, then the module won't be compatible with 1.8 anymore, and we'll only be testing latest/edge. wdyt about this scenario?

If we introduced a new relation tomorrow (I guess in `main`), then that would create the need for a new module. Now I 'm not sure how we would handle this exactly tbh. Would we created a new 1.8-compatible in `track/1.8` and update the one in `main` to be latest/edge-compatible? Would we keep only 1.8-compatible in `main`? It does feel a bit weird and is actually tied to the dicsussion comment above.

Reply:
since this is the `main` branch I agree with @orfeas-k to go with `latest/edge`
If we introduce a new relation in `main`, we must keep consistency with the terraform module, meaning that any new relation should take effect in `outputs.tf`. This is something we have to keep in mind from now on.

Reply:
Based in our discussion from yesterday, we should remove this and the name of the model and use the defaults from the workflow.